### DISCUSSION TITLE
[QUESTION] Is support add muti-video stream to the same peer connection？

### DISCUSSION THREAD

**Discussion by tplusy** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1901] [Q&A]
Hi

My hardware have two camera，I Just want to create only one peer connection and  send 2 (camera) video stream to the one peer connection, so in the peer have 2 viewer to display 2 video stream.

**Comment by niyatim23** [comment]
Hi @tplusy, you would need two peer connections for that. A single peer connection is used for a single video and an audio track

**Reply by tplusy** [reply]
Do you have any plans to implement this feature?
